packages/@aws-cdk/cx-api/lib/cloud-artifact.ts

```diff
@@ -24,6 +24,13 @@ export interface AwsCloudFormationStackProperties {
    * @default - name derived from artifact ID
    */
   readonly stackName?: string;
+
+  /**
+   * Whether to enable termination protection for this stack.
+   *
+   * @default false
+   */
+  readonly terminationProtection?: boolean;
```

Comment:
This needs to be added to the cloud-assembly-schema module, right @iliapolo ?

Reply:
This relates to `cx-api` changes we want to introduce with this PR: http://github.com/aws/aws-cdk/pull/6366/.
Until then, it doesn't need to be added to `cloud-assembly-schema` because its just a part of the properties, which are defined as `any` in the schema.

Reply:
Ok, got it. for some reason I thought that artifacts are already strongly-typed in the schema...